styles/web-scraping.module.css

```diff
@@ -5,6 +5,12 @@
   grid-gap: 0.5rem;
 }
 
+@media only screen and (max-width: 400px) {
```

Comment:
Once we move this to GlobalStyles, let's use [theme.breakpoints](https://mui.com/material-ui/customization/breakpoints/) for this.

Reply:
Yes, once we move all of this to `sx` props.